listenbrainz/spark/request_manage.py

```diff
@@ -137,11 +137,26 @@ def request_dataframes(days):
     send_request_to_spark_cluster(_prepare_query_message('cf_recording.recommendations.create_dataframes', params=params))
 
 
+def parse_list(ctx, args):
+    return list(args)
+
+
 @cli.command(name='request_model')
-def request_model():
+@click.option("--rank", callback=parse_list, default=[5, 10], type=int, multiple=True, help="Number of hidden features.")
```

Comment:
These descriptions are not very useful for someone who doesn't really understand what is going on -- but, this is not the place to try and fix that. Would it be possible to print a link where the user can find more docs?

Reply:
https://spark.apache.org/docs/2.1.0/mllib-collaborative-filtering.html 

Looks good?